sycl/include/CL/sycl/device.hpp

```diff
@@ -195,6 +195,10 @@ class __SYCL_EXPORT device {
   /// \return true if the SYCL device has the given feature.
   bool has(aspect Aspect) const;
 
+  bool is_assert_fail_supported() const {
+    return has(aspect::ext_oneapi_native_assert);
+  }
+
```

Comment:
I do not think we should add public functions to the `device` class (or any SYCL class) that are not part of the SYCL 2020 specification and not part of any DPC++ extension.  Why do we need this function?  Can't you just call `device::has(aspect::ext_oneapi_native_assert)` instead?

Reply:
Fixed.

Reply:
Seems like the function is still there.  Did you forget to push a commit?

Reply:
Ah! Exactly.